KutokAccounting.Services/Vendors/Models/QueryParameters.cs

```diff
@@ -0,0 +1,3 @@
+﻿namespace KutokAccounting.Services.Vendors.DataTransferObjects;
+
+public sealed record QueryParameters(string? Name, string? SearchString, int Page = 1, int PageSize = 10);
```

Comment:
```suggestion
public sealed record QueryParameters(string? Name, string? SearchString, Pagination pagination);
```
Ну і відповідно дефолтні значення перенести до пагінації